### PR TITLE
UUID support

### PR DESCRIPTION
**Pull request by jacobbaungard**
This adds UUID support and merges it into the slim-poller-2.0 branch.

### CODE REVIEW

**Comment by jacobbaungard** [comment]
note: tests are still likely to fail, but that is a separate issue and happens on master as well.

**Review by erikdsjostrom — COMMENTED** [review]
This looks good to me, but perhaps someone with C-eyes should take a look.

File: shared/node.c [hunk 1]
```diff
@@ -856,6 +863,9 @@ int node_send(merlin_node *node, void *data, unsigned int len, int flags)
 	if (!node || node->sock < 0)
 		return 0;
 
+	strcpy(pkt->hdr.from_uuid, ipc.uuid);
+
+
```
Comment (jimorie) [inline shared/node.c:868]: One line too many?

**Review by jimorie — APPROVED** [review]
I can't really offer a qualified review either, but with that said this looks good to me. Found only a couple of typos that can be ignored.

**Comment by jacobbaungard** [comment]
Typos fixed

**Pull request merged**